src/server/db_slice.cc

```diff
@@ -1715,9 +1692,34 @@ void DbSlice::PerformDeletion(Iterator del_it, DbTable* table) {
 }
 
 void DbSlice::OnCbFinish() {
```

Comment:
Lets rename to OnCbFinishBlocking so it will be clear on the caller that this is a blocking function